utils/coreutils/cmdutils.go

```diff
@@ -120,6 +120,22 @@ func FindBooleanFlag(flagName string, args []string) (flagIndex int, flagValue b
 	return -1, false, nil
 }
 
+// Boolean flag can be provided in one of the following forms:
+// 1. --flag=value, where value can be true/false
+// 2. --flag, here the value is true
+// Return values:
+// flagIndex - index of flagName in args.
+// flagValue - value of flagName.
+// err - error if flag exists, but we failed to extract its value.
+// If flag does not exist flagIndex = -1 with true value and nil error.
```

Comment:
I think that the last line in the above description isn't completely clear. The same goes for the last line in the description of the ```FindBooleanFlag```.
Actually, I suggest that instead of duplicating most of the description for this function, we'll described it as - "The same as the FindBooleanFlag function, but ...".